bundle/src/main/java/com/adobe/acs/tools/csv_asset_importer/impl/Parameters.java

```diff
@@ -289,4 +293,18 @@ public String toString() {
 
         return sw.toString();
     }
-}
\ No newline at end of file
+
+	/**
+	 * @return the inFileMimeType
+	 */
+	public String getInFileMimeType() {
```

Comment:
not sure why this isn't just called `getMimeType()` but maybe there's a reason I'm missing.